assessment.md

```diff
@@ -143,39 +143,96 @@ TODO: add description for each assessment level for this skill.
 
 * Requirements engineering
 
-TODO: add description for each assessment level for this skill.
+Requirements engineering will be evaluated on this scale:
+
+* None = Fails to adds any requirements for the program and shows no knowledge of what the program is doing
+* Inadequate = Adds one requirement for the program
+* Adequate = Adds two requirements for the program
+* Good = Adds three requirements for the program
+* Excellent = Adds more than three requirements for the program and shows knowledge of what the program is doing
 
 * Software architecture
 
-TODO: add description for each assessment level for this skill.
+The architecture of the software will be evaluated on this scale:
+
+* None = The program demonstrates no understanding or implementation of software architecture.
+* Inadequate = The program displays experience with software architecture, but there are flow and design issues.
+* Adequate = The program displays experience and understanding of software architecture with minimal flow and design issues.
+* Good = the program displays an extensive knowledge of software architecture theory and underlying experience with no flow or design issues.
+* Excellent = the program displays an extensive knowledge of software theory and underlying experience as a means of conducting and managing the design/flow. Develops concepts and is able to help other team members with the architecture of their software.
```

Comment:
There is a small grammatical issue that is causing some confusion in this statement. It's not clear who `Develops concepts and is able to help other team members with the architecture of their software.` Are you referring to the program or the student?